theme/src/main/java/uk/gov/android/ui/themeM3/ThemeM3.kt

```diff
@@ -70,7 +79,27 @@ fun GdsThemeM3(
     MaterialTheme(
         colorScheme = colors,
         typography = TypographyM3,
-        shapes = ShapesM3,
-        content = content
-    )
+        shapes = ShapesM3
+    ) {
```

Comment:
Should we pass in the `typography` and `shapes` variables as default parameters to the `GdsThemeM3`?